akka-stream/src/main/mima-filters/2.6.x.backwards.excludes/30871-update-ssl-config-core.backwards.excludes

```diff
@@ -0,0 +1,2 @@
+# internal
+ProblemFilters.exclude[DirectMissingMethodProblem]("com.typesafe.sslconfig.akka.AkkaSSLConfig.validateDefaultTrustManager")
```

Comment:
That looks like public API to me and not internal (deprecated since 2.6.0 though) so this is breaking our binary compatibility guarantee as far as I can see.

Reply:
OK, I had a look at https://github.com/lightbend/ssl-config/pull/302. For Java 8+ the algorithms and cipher checking isn't needed anymore, that's why @wsargent removed it from the ssl-config library.
If you look at the removed `validateDefaultTrustManager` method in this pull request it uses the removed `disabledKeyAlgorithms` list from ssl-config and checks if there any certificates that matches. As far as I understand In Java 8+ this isn't relevant anymore since it doesn't ship with "bad" certificates anymore, so IMHO this `validateDefaultTrustManager` method is basically useless since akka requires Java 8.

To not break binary compatibly, what about, instead of removing the method, make it no-op
```scala

def validateDefaultTrustManager(sslConfig: SSLConfigSettings): Unit = {
  // Doing nothing anymore, was useful only in Java <=7
}
```
?

Reply:
BTW, the exact same `validateDefaultTrustManager` got removed in https://github.com/playframework/play-ws/pull/590 already, which got approved by Will Sargent as well, so I am pretty sure this method isn't relevant anymore.

Reply:
Or maybe
```scala
def validateDefaultTrustManager(sslConfig: SSLConfigSettings): Unit = {
  log.warning("validateDefaultTrustManager is not doing anything since akka 2.6.19, it was useful only in Java 7 and below");
}
```
To be honest, I highly doubt anyone is using this method.